### DISCUSSION TITLE
Add paleo fields to label editor's visual interface

### DISCUSSION THREAD

**Discussion by ljwalker** [Symbiota/community-board#235] [Wish List]
Fields from the paleo module are currently excluded from the label editor's visual interface:

![Screen Shot 2022-10-27 at 10 24 47 AM](https://user-images.githubusercontent.com/11181096/198357633-0ee6bc3e-b95f-45ac-a6a2-be395e2fcd4b.png)

**Comment by skswen** [comment]
This would be an extremely useful feature for us at the DMNS. If these fields were added, we could potentially use Symbiota as our main source for label creation, which would be significantly simpler and more consistent than our current process.
